code/modules/paperwork/photocopier.dm

```diff
@@ -13,13 +14,23 @@
 	power_channel = EQUIP
 	max_integrity = 300
 	integrity_failure = 100
-	var/emag_cooldown
+	var/copying = FALSE //are we copying right now?
 	atom_say_verb = "bleeps"
+
 	var/obj/item/copyitem = null	//what's in the copier!
+	var/obj/item/folder = null   //Folder in the copier
+
 	var/copies = 1	//how many copies to print!
 	var/toner = 60 //how much toner is left! woooooo~
+	var/toner_requirement = 1 //How much toner will be required for this operation
 	var/maxcopies = 10	//how many copies can be copied at once- idea shamelessly stolen from bs12's copier!
-	var/mob/living/ass = null
+	var/mob/living/M = null //Whose ass are we copying? This is important information, hence why there is a var for it.
```

Comment:
Please name this something more descriptive, something other than `M`

Reply:
changed to copy_mob